src/amp/components/WithAds.tsx

```diff
@@ -49,6 +49,9 @@ export const WithAds = ({ items, adSlots, adClassName, adInfo }: Props) => {
 			return (
 				<>
 					{item}
+					{/* TODO: we should not be assuming the JSX has an ID property and using it as such, we should update `items` to contain IDs and JSX elements */}
+					{/*
+                    // @ts-ignore */}
```

Comment:
!!!! 🔴   This will need to be fixed in a future PR (there is already a lot happening here) 🔴  !!!!

We are accessing `id` as a prop from a JSX Element. I am not sure why this currently works, but we should instead send down an array continaing `ID` and `JSX.Elements[]`